README.md

```diff
@@ -0,0 +1,92 @@
+# mptcp-check
+A website to check the validity of a mptcp connection
+
+The website is build as a Flask app behind a lighttpd server.
+Everything is package in a docker container.
+
+## installing
+you will need to create the too following scripts. I would recommend the following. Do not forget to replace the `email` and `url`
+
+`start.sh`:
+``` bash
+#!/bin/bash
+email= #the email used by certbot
+url= #your domain name
+path=/var/docker/mptcp-check
+
+docker run --rm --name certbot-init \
+            -v "$path/cert:/etc/letsencrypt:rw" \
+            -p 80:80 \
+            certbot/certbot certonly \
+                --non-interactive --agree-tos \
+                --email $email \
+                -d $url \
+                --standalone
+
+docker run --name mptcp-check \
+            -v "$path/www:/var/www/:ro" \
+            -v "$path/cert/:/etc/letsencrypt/:ro" \
+            -p 80:80 -p 443:443 \
```

Comment:
did you check the port 80 was OK?

Reply:
it seems to only not work with curl, I have no clue why.

Reply:
I know that with a `.dev`, the upper layer forces to use `https`: https://ma.ttias.be/chrome-force-dev-domains-https-via-preloaded-hsts/

So maybe the browsers don't even try to use the port 80?

Here, we have an ICMP error, it looks like there is an issue with the firewall:

```
15:54:15.625189 IP 192.168.178.52.52850 > 5.196.67.207.80: Flags [S], seq 4252628027, win 32120, options [mss 1460,sackOK,TS val 1612797082 ecr 0,nop,wscale 7], length 0
15:54:15.665430 IP 5.196.67.207 > 192.168.178.52: ICMP 5.196.67.207 tcp port 80 unreachable, length 68
```

Reply:
since it is not related to the configuration, I will close the conversation.